### PR TITLE
move hello.txt to the guide

### PR DESCRIPTION
**Pull request by nslater**
https://trello.com/c/LSPyawQu/1

### CODE REVIEW

**Comment by nslater** [comment]
see crate/guide#13

**Comment by nslater** [comment]
[Move Hello CrateDB to right place](https://trello.com/c/LSPyawQu/1-move-hello-cratedb-to-right-place)

File: blackbox/docs/installation.txt [hunk 1]
```diff
@@ -94,7 +94,8 @@ additional packages if we see enough interest.
 
  :doc:`Configuring CrateDB <configuration>`
 
- :doc:`Hello CrateDB <hello>` - To learn how to interact with CrateDB.
+ `Hello CrateDB <https://crate.io/docs/crate/guide/tutorials/hello.html>`_ - To
```
Comment (Autophagy) [inline blackbox/docs/installation.txt]: Could you change this link to `` `Hello CrateDB`_ `` and put at `` .. _`Hello CrateDB`: https://crate.io/docs/crate/guide/tutorials/hello.html `` at the bottom of the document, to be consistent with the other external links in the document?

File: blackbox/docs/index_html.rst [hunk 2]
```diff
@@ -4,7 +4,8 @@ First Steps
 
 - In order to get CrateDB up and running take a look at the :doc:`installation`.
 
-- If you are new to CrateDB, it is probably the best to say :doc:`hello`.
+- If you are new to CrateDB, it is probably the best to say `hello
```
Comment (Autophagy) [inline blackbox/docs/index_html.rst]: See my other comment: same deal here :slightly_smiling_face:

File: blackbox/docs/installation.txt [hunk 3]
```diff
@@ -108,3 +107,4 @@ additional packages if we see enough interest.
 .. _`docker-crate`: https://github.com/crate/docker-crate/
 .. _`Multi Node Setup`: https://crate.io/docs/scale/multi_node_setup/
 .. _`Java 8`: http://www.oracle.com/technetwork/java/javase/downloads/index.html
+.. _Hello CrateDB: https://crate.io/docs/crate/guide/tutorials/hello.html
```
Comment (Autophagy) [inline blackbox/docs/installation.txt]: Multiple word links like "Hello CrateDB" have to be wrapped in backticks, I think

Comment (Autophagy) [inline blackbox/docs/installation.txt]: Oh, I think im wrong actually, I dont think it matters.

**Pull request merged**